docs/advanced/settings.md

```diff
@@ -424,6 +426,99 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
+kind: Setting
+metadata:
+  name: support-bundle-image
+...
+status: {}
+value: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.36","imagePullPolicy":"IfNotPresent"}'  // your setting value
+```
+
+Later this cluster is upgraded, it is further like:
+
+```
+apiVersion: harvesterhci.io/v1beta1
+default: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.38","imagePullPolicy":"IfNotPresent"}'  // default value, automatically updated after upgrade
+kind: Setting
+metadata:
+  name: support-bundle-image
+...
+status: {}
+value: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.36","imagePullPolicy":"IfNotPresent"}'  // your setting value is kept unchanged
+```
+
+Now the set `value` has image tag `v0.0.36` but the default image tag is `v0.0.38`, it is outdated.
+
+To clear the customized setting and stick to the default value, you need to operate with `kubectl`, then remove the whole line of `value` and save.
+
+```
+$ kubectl edit settings.harvesterhci.io support-bundle-image
+```
+
+After clearing the customized setting, it is like:
+
+```
+apiVersion: harvesterhci.io/v1beta1
+default: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.38","imagePullPolicy":"IfNotPresent"}'
+kind: Setting
+metadata:
+  name: support-bundle-image
+...
+status: {}
+```
+
+For this setting, the UI menu **Use the default value** copies the `default` to the `value` field.
+
+![](/img/v1.3/advanced/support-bundle-image-set-use-default-value.png)
+
+After clicking **Save**, the object is updated to:
+
+```
+apiVersion: harvesterhci.io/v1beta1
+default: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.38","imagePullPolicy":"IfNotPresent"}'  // default
+kind: Setting
+metadata:
+  name: support-bundle-image
+...
+status: {}
+value: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.38","imagePullPolicy":"IfNotPresent"}'  // copied from default
+```
+
+In the future when you upgrade the cluster to a new version the `default` image tag may be like `v0.0.40`, the `value` field will be outdated again.
+
+:::note
+
+- Harvester always sets the `default` field to the image tag which is packed into Harvester ISO. When the cluster is upgraded, the `default` field is also updated automatically. As long as the `value` field is not set or empty, Harvester sticks to the `default` field. It makes sure the image is on the cluster and is up-to-date.
+
+- In general you do not need to set this setting. If you plan to set a specific image, be aware that it will take precedence of the `default` and is not updated automatically.
+
+- If you have set a customized image but want to follow the default image, then use the `kubectl` command to clear the `value` field.
+
+- The UI menu **Use the default value** is not the best option to clear this setting. It copies the current `default` to `value`, the `default` will be updated when upgrading but the `value` field will not.
```

Comment:
cc @torchiaf @a110605 @bk201 

It is a bit surprised to observe the side affect of `Use the default value` in this context. UI? copies the `default` field to `Value` field.

Alternatively, if user clicks `Use the default value` and does not further change the configuration, when `saving`, UI deletes the `value` field; thus it really always sticks to `default`.

In different Harvester releases, Harvester may change the `default`.

WDYT? thanks.